app/utils/timezone.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {isMinimumServerVersion} from 'mattermost-redux/utils/helpers';
+import {Client4} from 'mattermost-redux/client';
+
+import DeviceInfo from 'react-native-device-info';
+
+export function getDeviceTimezone() {
+    return DeviceInfo.getTimezone();
+}
+
+export function isTimezoneSupportEnabled() {
+    return isMinimumServerVersion(Client4.getServerVersion(), 4, 9);
```

Comment:
@hmhealey sure.

Although does it make sense to put this in redux?  My understanding is that the webapp does not check for this since it's always in sync with the server.

Reply:
Miguel he is talking about the server version thatbis available in entities.general.serverVersion from the redux store instead of using the Client4 function to retrieve it

Reply:
gotcha 👍